components/os/macos_servicemanagers.go

```diff
@@ -0,0 +1,27 @@
+package os
+
+import (
+	"fmt"
+
+	"github.com/DataDog/test-infra-definitions/common/config"
+	"github.com/DataDog/test-infra-definitions/components/command"
+	"github.com/pulumi/pulumi-command/sdk/go/command/remote"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+type macOSServiceManager struct {
+	e      config.CommonEnvironment
+	runner *command.Runner
+}
+
+func newMacOSServiceManager(e config.CommonEnvironment, runner *command.Runner) ServiceManager {
+	return &macOSServiceManager{e: e, runner: runner}
+}
+
+func (s *macOSServiceManager) EnsureRunning(serviceName string, triggers pulumi.ArrayInput, opts ...pulumi.ResourceOption) (*remote.Command, error) {
```

Comment:
It may `Start` or `Restart`, I tried to find a name that express the desired state (as that's we want to do with Pulumi). I may need to think of a better naming but I don't think `Restart` expresses that.